target/linux/ath79/patches-6.1/317-MIPS-pci-ar724x-unset-power-down-of-pll-on-AR934x.patch

```diff
@@ -0,0 +1,24 @@
+--- a/arch/mips/include/asm/mach-ath79/ar71xx_regs.h
++++ b/arch/mips/include/asm/mach-ath79/ar71xx_regs.h
+@@ -347,6 +347,8 @@
+ #define AR934X_PLL_CPU_DDR_CLK_CTRL_DDRCLK_FROM_DDRPLL	BIT(21)
+ #define AR934X_PLL_CPU_DDR_CLK_CTRL_AHBCLK_FROM_DDRPLL	BIT(24)
+ 
++#define AR934X_PLL_PCIE_CONFIG_PLL_PWD		BIT(30)
++
+ #define AR934X_PLL_SWITCH_CLOCK_CONTROL_MDIO_CLK_SEL	BIT(6)
+ 
+ #define QCA953X_PLL_CPU_CONFIG_REG		0x00
+--- a/arch/mips/pci/pci-ar724x.c
++++ b/arch/mips/pci/pci-ar724x.c
+@@ -360,7 +360,9 @@ static void ar724x_pci_hw_init(struct ar
+ 	} else {
+ 		/* remove the reset of the PCIE PLL */
+ 		ppl = ath79_pll_rr(AR724X_PLL_REG_PCIE_CONFIG);
+-		ppl &= ~AR724X_PLL_REG_PCIE_CONFIG_PPL_RESET;
++		ppl &= of_device_is_compatible(apc->np, "qcom,ar9340-pci") ?
++			~AR934X_PLL_PCIE_CONFIG_PLL_PWD :
++			~AR724X_PLL_REG_PCIE_CONFIG_PPL_RESET;
```

Comment:
It's safe to use `ppl &= ~(AR934X_PLL_PCIE_CONFIG_PLL_PWD | AR724X_PLL_REG_PCIE_CONFIG_PPL_RESET)` for all SoCs.

Reply:
OK, I'll update...